common/src/main/java/io/netty/util/internal/GraalDetector.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.util.internal;
+
+abstract class GraalDetector {
```

Comment:
@rpuch can we merge this directly to `PlatformDependent0` ? Seems overkill to add this class just for exposing one method etc

Reply:
I've moved the code to `PlatformDependent0` and removed `GraalDetector` class. Please let me know if it is ok now.